omod-1.8/src/test/resources/create_patient_from_get.json

```diff
@@ -0,0 +1,267 @@
+{
+    "uuid":"a65fc361-f6ac-40ba-8485-94749c061509",
+    "display":"id-B - Jonnea Bijo",
+    "identifiers":[
+        {
+            "display":"TEST Primary Identifier type = id-B",
+            "uuid":"035be43e-a902-48bc-a22c-0ad01aaff731",
+            "identifier":"id-B",
+            "identifierType":{
+                "uuid":"1a339fe9-38bc-4ab3-b180-320988c0b968",
+                "display":"TEST Primary Identifier type - ingored",
+                "links":[
+                    {
+                        "rel":"self",
+                        "uri":"http://192.168.1.100/openmrs/ws/rest/v1/patientidentifiertype/0t504478-1701-49da-9d8b-3e1d12ab6c5b"
+                    }
+                ]
+            },
+            "location":{
+                "uuid":"8d6c993e-c2cc-11de-8d13-0010c6dffd0f",
+                "display":"Outpatient Clinic",
+                "links":[
+                    {
+                        "rel":"self",
+                        "uri":"http://192.168.1.100/openmrs/ws/rest/v1/location/58c57d25-8d39-41ab-8422-108a0c277d98"
+                    }
+                ]
+            },
+            "preferred":true,
+            "voided":false,
+            "links":[
+                {
+                    "rel":"self",
+                    "uri":"http://192.168.1.100/openmrs/ws/rest/v1/patient/a65fc361-f6ac-40ba-8485-94749c061509/identifier/035be43e-a902-48bc-a22c-0ad01aaff731"
+                },
+                {
+                    "rel":"full",
+                    "uri":"http://192.168.1.100/openmrs/ws/rest/v1/patient/a65fc361-f6ac-40ba-8485-94749c061509/identifier/035be43e-a902-48bc-a22c-0ad01aaff731?v=full"
+                }
+            ],
+            "resourceVersion":"1.8"
+        }
+    ],
+    "person":{
+        "uuid":"a65fc361-f6ac-40ba-8485-94749c061509",
+        "display":"Jonnea Bijo",
+        "gender":"M",
+        "age":23,
+        "birthdate":"1995-01-01T00:00:00.000+0000",
+        "birthdateEstimated":true,
```

Comment:
This was added in 1.11 so it should not be here, it's one of the reasons why the test fails since it is a 1.9 test